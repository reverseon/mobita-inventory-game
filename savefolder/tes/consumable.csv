id;nama;deskripsi;jumlah;rarity
C5;Simping Potion;Make you go higher while simp;20;S
C20;sdfjskdh;fdskjfhdsjh;300;S