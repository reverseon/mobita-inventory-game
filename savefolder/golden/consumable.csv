id;nama;deskripsi;jumlah;rarity
C5;Simping Potion;Make you go higher while simp;20;S
C8;Gambling Potion;Luck++;0;A
C1;Hyperpotion;Restore HP +200;100;A
C2;Grad Potion;Restore HP +50;20;B
C3;MP Potion;Restore Mana +200;0;A
C4;EXTRA MP POTION;Restore MP Potion +400;2;S
